dtfs-central-api/src/v1/controllers/utilisation-report-service/get-utilisation-reports.controller/helpers.test.ts

```diff
@@ -0,0 +1,43 @@
+import { parseReportPeriod } from './helpers';
+
+describe('get-utilisation-reports.controller helper', () => {
+  describe('parseReportPeriod', () => {
+    it('returns undefined if the input is undefined', () => {
+      // Act
+      const result = parseReportPeriod(undefined);
+
+      // Assert
+      expect(result).toBeUndefined();
+    });
+
+    it('returns undefined if the parsed report period is not a valid report period', () => {
```

Comment:
I think the expected behaviour of this should be:
- if `reportPeriod` is `undefined`, return `undefined`
- if `reportPeriod` is defined but cannot be parsed OR `JSON.parse` fails, it should throw an error

I am planning on adding this change, but won't push until other reviews are finished